test/uk/gov/hmrc/gform/formtemplate/FormTemplatesControllerRequestHandlerTest.scala

```diff
@@ -789,4 +939,186 @@ class FormTemplatesControllerRequestHandlerTest extends WordSpec with Matchers w
        |    "fields": []
        |  }
        |}""".stripMargin
```

Comment:
all the valid and invalid json methods below could be simplified, to a single json and just replacing the dmsFormId part with a passed in value, which could be an Option. Reduces a lot of copy paste also makes it simpole to updated the templates change in future

Reply:
Yes. Changed.